cmd/cmp/main_test.go

```diff
@@ -2,32 +2,124 @@ package cmp
 
 import (
 	"fmt"
-	"github.com/stretchr/testify/assert"
 	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
 	"testing"
 )
 
 func TestRun(t *testing.T) {
 
 	FileIdRegexStr = `.*(query_\d{2})(?:_c0)?\.output`
 
-	expectedReturns := []int{0, 1, 1, 1, 0, 0}
+	execPath, err := os.Executable()
+	fmt.Println(execPath)
+
+	if err != nil {
+		// Handle error
+	}
+	execDir := filepath.Dir(execPath)
+	for i := 0; i <= 5; i++ {
+		outputDir := filepath.Join(execDir, fmt.Sprintf("test_%d_output", i))
+		err := os.Mkdir(outputDir, 0755)
+		if err != nil {
+			// Handle error
```

Comment:
What are we trying to do here? creating directory then delete?

Reply:
Sorry, needed to clean that up. I was testing permissions and directory creation.